tensorflow/python/ops/math_ops.py

```diff
@@ -288,7 +288,10 @@ def argmax_v2(input, axis=None, output_type=dtypes.int64, name=None):
   <tf.Tensor: shape=(), dtype=int64, numpy=0>
 
   Args:
-    input: A `Tensor`.
+    input: A `Tensor`. Must be one of the following types: `float32`, `float64`,
```

Comment:
I would rather not list out types here unless we have a valid reason for restricting them.

Otherwise, it will lead to more documentation mismatches like the ones you're trying to fix here.  If someone does add support, the documentation will be out-of-sync.